action.yml

```diff
@@ -43,7 +44,7 @@ outputs:
     description: The error and warning messages for each one of the analyzed commits
 runs:
   using: docker
-  image: docker://wagoid/commitlint-github-action:5.4.0
+  image: Dockerfile
```

Comment:
Did you add `Dockerfile` to test in your fork? The `commitlint` workflow does this too, it replaces `image` field with `Dockerfile` as part of the steps: https://github.com/wagoid/commitlint-github-action/blob/master/.github/workflows/commitlint.yml#L11